app/javascript/components/student/TracksList.jsx

```diff
@@ -28,12 +28,19 @@ export function TracksList({ statusOptions, ...props }) {
   )
 
   return (
-    <div className="student-track-list">
-      <Search dispatch={dispatch} />
-      <StatusFilter dispatch={dispatch} options={statusOptions} />
-      {isLoading && <Loading />}
-      {isError && <p>Something went wrong</p>}
-      {isSuccess && <List data={data} />}
+    <div className="c-tracks-list">
+      <div className="search-bar">
```

Comment:
Do you think we should put this div within the `<Search />` component?

Reply:
This div will also have the filters etc within it, so it can't. It's potentially badly named though - as it covers search and filter, not just search. I might just inline the styles into it rather than have a named class for it. I'm trying to work out when to do which of those things still.

Reply:
Cool!